javascript/ql/experimental/adaptivethreatmodeling/test/generic_feature_testing/NonTotalFeature.ql

```diff
@@ -0,0 +1,8 @@
+import javascript
+import experimental.adaptivethreatmodeling.EndpointFeatures
+import TestUtil
+
+// every feature should produce a value for all endpoints
```

Comment:
I don't think this should be true.  We might have features that don't apply to certain endpoints.  For example, `enclosingFunctionBody` doesn't make sense if an endpoint is not within a function body.